code/gradle/release.gradle

```diff
@@ -8,7 +8,7 @@
  *
  * Release script
  * 1. gradle prepareRelease slowtest
- *   a.	Update version � remove snapshot
+ *   a.	Update version � remove snapshot
```

Comment:
Minor typo looks like you have an emdash or dash which displays as the weird question mark. The minus sign will display properly. This is repeated on line 20 as well.